tell/match.py

```diff
@@ -175,7 +175,10 @@ def prepare_data(fips_file, service_area_file, sales_ult_file, bal_auth_file):
 
     # read in data
     df_fips = pd.read_csv(fips_file)
-    df_states = pd.read_excel(service_area_file, sheet_name='Counties_States')
+    try:
+        df_states = pd.read_excel(service_area_file, sheet_name='Counties')
+    except:
+        df_states = pd.read_excel(service_area_file, sheet_name='Counties_States')
```

Comment:
Here was that inconsistency i mentioned in Slack... probably better to fix the data source if that's under our control -- if not, might want to make my quick fix a little more robus

Reply:
I think it was the raw data source but I am waiting to hear from Casey B about. I will merge it for now in case that is the situation.